new/LIT6402MJInfancy.xml

```diff
@@ -0,0 +1,91 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6402MJInfancy" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">The Infancy Gospel</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="RL"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="RL" when="2021-03-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6402MJInfancy" passive="LIT2382Taamme"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+              <bibl>
+                <ptr target="bm:RompayArras1975Miracles"/><citedRange unit="item">44</citedRange>
+              </bibl>
+            </div>
+            <div type="edition">
+              <note>The following division into textparts and excerpts are taken from <ref type="mss" corresp="EMIP02392"/>,
+                  without subscription and supplications.</note>
+                  <label>The second recension of the Infancy Gospel</label>
```

Comment:
According to the catalogue this is 'The second recension of the Infancy Gospel. Van Rompay and Arras 1975, no. 44.' which is broken down into three parts, Jesus rides a ray of sunshine, Jesus makes birds out of mud and orders them to fly away, Jesus mixes and separates the colors of his master's ink

Reply:
Having only read the EAe article on the topic, it is not completely clear to me what is meant by this...In case you know it better and can quickly clarify it in a few words, that would be helpful, otherwise I think we can go ahead and trust the catalogue.